include/ccl_core.h

```diff
@@ -193,96 +179,21 @@ ccl_cosmology * ccl_cosmology_create_with_lcdm_params(
- * For specific cases see documentation for ccl_error.c
- * @return void
- */
-ccl_parameters ccl_parameters_create_flat_lcdm_bar(double Omega_c, double Omega_b, double h,
-						   double norm_pk, double n_s, double bcm_log10Mc,
-						   double bcm_etab, double bcm_ks, int *status);
-
-/**
- * Create a flat wCDM cosmology
- * @param Omega_c Omega_c 
- * @param Omega_b Omega_b 
- * @param w0 Dark energy EoS parameter
- * @param h Hubble constant in units of 100 km/s/Mpc
- * @param norm_pk the normalization of the power spectrum, either A_s or sigma8
- * @param n_s the power-law index of the power spectrum
- * @param status Status flag. 0 if there are no errors, nonzero otherwise.
- * For specific cases see documentation for ccl_error.c
- * @return void
- */
-ccl_parameters ccl_parameters_create_flat_wcdm(double Omega_c, double Omega_b, double w0, double h, double norm_pk, double n_s, int *status);
+                     double Neff, double* mnu, ccl_mnu_convention mnu_type,
+                     double w0, double wa, double h, double norm_pk,
+                     double n_s, double bcm_log10Mc, double bcm_etab, double bcm_ks,
+                     int nz_mgrowth,double *zarr_mgrowth,
+                     double *dfarr_mgrowth, int *status);
 
-/**
- * Create a flat waCDM cosmology
- * @param Omega_c Omega_c 
- * @param Omega_b Omega_b 
- * @param w0 Dark energy EoS parameter
- * @param wa Dark energy EoS parameter
- * @param h Hubble constant in units of 100 km/s/Mpc
- * @param norm_pk the normalization of the power spectrum, either A_s or sigma8
- * @param n_s the power-law index of the power spectrum
- * @param status Status flag. 0 if there are no errors, nonzero otherwise.
- * For specific cases see documentation for ccl_error.c
- * @return void
- */
-ccl_parameters ccl_parameters_create_flat_wacdm(double Omega_c, double Omega_b, double w0,double wa, double h, double norm_pk, double n_s, int *status);
 
-/**
- * Create an LCDM cosmology with curvature
- * @param Omega_c Omega_c 
- * @param Omega_b Omega_b 
- * @param Omega_k Omega_k 
- * @param w0 Dark energy EoS parameter
- * @param wa Dark energy EoS parameter
- * @param h Hubble constant in units of 100 km/s/Mpc
- * @param norm_pk the normalization of the power spectrum, either A_s or sigma8
- * @param n_s the power-law index of the power spectrum
- * @param status Status flag. 0 if there are no errors, nonzero otherwise.
- * For specific cases see documentation for ccl_error.c
- * @return void
- */
-ccl_parameters ccl_parameters_create_lcdm(double Omega_c, double Omega_b, double Omega_k, double h, double norm_pk, double n_s, int *status);
+/* ------- ROUTINE: ccl_parameters_create_flat_lcdm --------
+INPUT: some cosmological parameters needed to create a flat LCDM model
+TASK: call ccl_parameters_create to produce an LCDM model
+*/
+ccl_parameters ccl_parameters_create_flat_lcdm(double Omega_c, double Omega_b, double h,
```

Comment:
So we've only kept this one out of all the specific cosmologies?

Reply:
yes.